lib/sessions.js

```diff
@@ -44,18 +44,16 @@ module.exports = (app, config) => {
   app.use(cookieParser(config.session.secret, {
     path: '/',
     httpOnly: true,
-    secure: true
+    secure: config.protocol === 'https'
   }));
 
-  const sessionOpts = Object.assign({}, {
+  const sessionOpts = Object.assign({
     store,
-    cookie: {secure: true},
-    key: '',
```

Comment:
Looks as if it might now be called `name` https://github.com/expressjs/session/blob/master/index.js#L89